### ISSUE TITLE
no-irregular-whitespace reports a violation on the word "import" inside a template string

### ISSUE THREAD

**Issue by HamletDRC** [palantir/tslint#3590]
### Bug Report

- __TSLint version__: 5.8.0
- __TypeScript version__: 2.6.2
- __Running TSLint via__: (pick one) grunt-tslint

#### TypeScript code being linted

```ts
        const script : string = `
            ﻿import OfficeApp from 'OfficeApp';
            ﻿import OfficeApp from 'common/OfficeApp';
        `;
```

with `tslint.json` configuration:

```json
"no-irregular-whitespace": true
```

#### Actual behavior
>> ERROR: (no-irregular-whitespace) src/tests/NoRelativeImportsRuleTests.ts[20, 13]: Irregular whitespace not allowed

#### Expected behavior
This import is inside a template string, so I don't think a lint error should occur.

**Comment by ajafff** [comment]
The rule just scans though the whole file with a simple regex looking for irregular whitespace characters.
This has nothing to do with the `import` inside the template string. There must be an irregular whitespace character, probably from copy and past.

Also note that the rule also complains about irregular whitespace inside strings and comments.
